cross/cloudflared/digests

```diff
@@ -1,3 +1,3 @@
-cloudflared-2024.10.1.tar.gz SHA1 64a76aa90b6b37592a953f75848fb4fd440cffbf
-cloudflared-2024.10.1.tar.gz SHA256 7e35e3e57a65f8914c5b53896cfa3711153af78b95d971791602c6624d53a1e1
-cloudflared-2024.10.1.tar.gz MD5 364287f2daf356aa54e5da8af2cbcafc
+cloudflared-2025.1.0.tar.gz SHA1 754e85783eab60a3f7644b0ea7f6501066c781c7
+cloudflared-2025.1.0.tar.gz SHA256 f9223cdefaa4b75aa1c49638936af9d5007b6c7bd943ab70203bb75bf32467da
+cloudflared-2025.1.0.tar.gz MD5 0fd57ebebeadd3dd44fc2c1d6069ca15
```

Comment:
please keep the line endings.
you can update the `digests` file with `make clean` and `make digests` in the cross/cloudflard folder.